src/Process/3dTilesProcessing.js

```diff
@@ -80,6 +80,10 @@ export function $3dTilesCulling(node, camera) {
 }
 
 export function pre3dTilesUpdate(context, layer) {
```

Comment:
It's unfortunate that we can only clean when needsUpdate is true, and not in iTowns' downtime. But this would require some architectural changes.